create_infotexts.py

```diff
@@ -367,8 +369,8 @@ def process_depicted_place(self, place_string, places_mapping):
                 place_as_wikitext = place_string
 
             # Don't forget to add the commons categories, even though only wikidata is used in depicted people field
-                if places_mapping[place_string].get('commonscat'):
-                    self.content_cats.append(places_mapping[place_string]["commonscat"])
+            if places_mapping[place_string].get('commonscat'):
```

Comment:
OK, created [new task](https://phabricator.wikimedia.org/T164186) not to forget it